### ISSUE TITLE
Wait: allow expression to allow dynamic calculation of wait time

### ISSUE THREAD

**Issue by tomberek** [bigskysoftware/_hyperscript#16]
Something like:
```
<button _="on click wait #input.value then toggle .clicked on me">Click</button>
```

**Comment by benpate** [comment]
I think this works now -- [check out this CodePen](https://codepen.io/benpate/pen/YzpOoxB?editors=1111).  Is it OK to close this?

For the record, `wait` is now super-powerful because it can wait on individual events, too.

**Comment by tomberek** [comment]
Works.

**Comment by mfeif** [comment]
I stumbled upon this too; I have a question for those of you who are interested in this feature.

I see the codepen; that's fantastic.

Is there a way for the time expression to be an evaluated expression? Rather than pulling the value from the dom, could it be some variable?

Say you pulled a status object from a server that includes a recommended wait time; as far as I can tell, there is no way to plug that value into the wait command. I suppose I could stick it in a hidden form and then get that from the hidden form, but I'd prefer a more direct way.

TLDR: how can I do this?
```
wait `${waitTime}s`
```

**Comment by benpate** [comment]
The "s" is actually interpreted as an operator (which really just multiplies the previous number by 1000).  So, treat it like a number and you should be good.  This works for me: 

```html
<input id="myInput">
<button _="on click wait (#myInput.value s) then log me">Click</button>
```
